components/FactModal/index.jsx

```diff
@@ -26,9 +26,12 @@ export default function FactModal({ isLoading }) {
 
   useEffect(() => {
     let intervalId = null;
+    let factIndex = 0;
+
     if (isLoading) {
       intervalId = setInterval(() => {
-        setCurrentFactIndex(prev => (prev === facts.length - 1 ? 0 : prev + 1));
```

Comment:
This is much more readable and I love the refactor Anna! However, this makes me think perhaps we could get by with reusing the `currentFactIndex` state that is already defined on line 25. What do you think about something like this? 

```
intervalId = setInterval(() => {
      setCurrentFactIndex((currentFactIndex + 1) % facts.length);
}, seconds(5));
```

Reply:
done!